### ISSUE TITLE
(epub-test-fail: pub-foreign_image)

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader-epub-tests#23]
The [pub-foreign_image](https://w3c.github.io/epub-tests/index.html#pub-foreign_image) test fails at 
* Test passes if you see a snowflake image below.

_Snowflake image is not displayed, alt texts indicates : test fail_

[OPF manifest fallback](https://github.com/w3c/epub-tests/blob/03d3b84722cb0f5c3439702c38e8b354f8f33e31/tests/pub-foreign_image/EPUB/package.opf#L22-L23)
```
<item id="image-tiff" href="p009-star.psd" media-type="image/psd" fallback="image-png" />
<item id="image-png" href="Snowflake.png" media-type="image/png" />
```
![Test table line 115](https://github.com/edrlab/thorium-reader-epub-tests/blob/main/Test_Thorium_2-0.md#115)

**Comment by panaC** [comment]
fallback is not implemented ?
